src/browser/modules/EditorInfo/DisplayRelationshipDirection.jsx

```diff
@@ -68,8 +75,16 @@ function DisplayRelationshipDirection (props) {
             }}
             value={selectedDirection}
             onChange={e => {
-              setSelectedDirection(e.target.value)
-              setButtonVisibility(true)
+              if (
+                e &&
+                e.target.value !== propsToStateMap(props.relationshipEndpoint)
+              ) {
+                setSelectedDirection(e.target.value)
+                setButtonVisibility(true)
+              } else {
+                setSelectedDirection(e.target.value)
```

Comment:
why to set selection direction if values are same (same = what value was set earlier and what value is about to set are same)